src/enforcer.js

```diff
@@ -572,13 +669,18 @@ export class TrustedTypesEnforcer {
    * @param {function(!Function, ...*)} functionBody The wrapper function.
    */
   wrapFunction_(object, name, functionBody) {
+    if (!object) return;
```

Comment:
The type system should be covering this case.

Reply:
Mhh, I've changed this to `nullable` as it's a bit easier to implement. Do you want me to reveert that and make this non-nullable?

Reply:
Done.